includes/class-vindi-api.php

```diff
@@ -681,9 +681,12 @@ public function get_charge($charge_id)
      */
     public function is_merchant_status_trial_or_sandbox()
     {
+        if ('yes' === $this->sandbox)
```

Comment:
Fica mais fácil de entender se for `$this->sandbox === 'yes'`.
`Essa variável é igual a isso?` é mais legível do que `isso é igual essa variável?`

Mas não acho que isso seja um impedimento para o approve! :)

Reply:
![image](https://user-images.githubusercontent.com/31661772/46560566-a3b07680-c8ca-11e8-87f2-94be3cbff3c1.png)